.github/workflows/integer_multi_bit_multi_gpu_benchmark.yml

```diff
@@ -24,6 +29,7 @@ env:
   SLACK_USERNAME: ${{ secrets.BOT_USERNAME }}
   SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
   FAST_BENCH: TRUE
```

Comment:
is FAST_BENCH still used in the benchmarks code ?

Reply:
Yes, it deals with the precision selection